abipy/abio/factories.py

```diff
@@ -1413,6 +1413,109 @@ def dfpt_from_gsinput(gs_inp, ph_ngqpt=None, qpoints=None, do_ddk=True, do_dde=T
     return multi
 
 
+def conduc_from_inputs(scf_input, nscf_input, tmesh, ddb_ngqpt, eph_ngqpt_fine, sigma_erange):
```

Comment:
We usually run eph calculations with boxcutmin 1.1 and mixprec 1 to decrease memory and reduce the wall-time required to apply H1. You may want to add these two arguments as optional and provide default values. Abinit default is boxcutmin 2 and mixprec 0 but here you can also opt for the fast settings.